arch/arm/core/cortex_m/fault.c

```diff
@@ -508,7 +509,7 @@ static u32_t UsageFault(const z_arch_esf_t *esf)
 		/* Stack Overflows are always reported as stack corruption
 		 * errors. Note that the built-in stack overflow mechanism
 		 * prevents the context area to be loaded on the stack upon
-		 * UsageFault exception entry. As a result, we cannot rely
+		 * usage_fault exception entry. As a result, we cannot rely
```

Comment:
This needs to be UsageFault - we are referring to the actual exception name, according to the ARM terminology